beamer/cli.py

```diff
@@ -51,6 +51,11 @@ def _sigint_handler(agent: Agent) -> None:
     type=int,
     help="Time in seconds to wait for a fill event before challenging a false claim.",
 )
+@click.option(
+    "--expiry-wait-time",
+    type=int,
+    help="Time left until expiry in seconds to fill a request.",
```

Comment:
I find the `expiry-wait-time` naming confusing and would prefer if we came up with a clearer name.
The way I was thinking about this:
```
|---------|--------------------------------|
A    T1   B                 T2             C
```
where 
```
A = request creation
B = earliest moment an agent can fill the request
C = request expiration
```

If we were interested in period `T1`, then it would be easier because that's easier to express. E.g. `fill-age` i.e. the age at which the request can be filled by the agent. However, I don't have a nice naming yet for `T2`.

Reply:
Here was some missunderstanding going on. The value describes the time left, a request must be still valid. If the a request es valid for less then the added value then the agent will ignore the request. So according to your drawing: 



```
|---------|--------------------------------|
A    T1   B                 T2             C
```
where 
```
A = request creation
B = last moment an agent is willing to fill the request
C = request expiration
```

Reply:
unsafe-fill-time @fredo @istankovic ?

Reply:
I think `unsafe-fill-time` is definitely an improvement over the current name.